packages/manager/modules/dedicated-universe-components/src/directives/sshkeySwitch/index.js

```diff
@@ -0,0 +1,10 @@
+import angular from 'angular';
+
+import sshKeySwitchDirective from './sshkeySwitch';
+import './sshkeySwitch.css';
+
+const moduleName = 'ducSshkeySwitch';
+
+angular.module(moduleName, []).directive('sshkeySwitch', sshKeySwitchDirective);
```

Comment:
```suggestion
angular.module(moduleName, []).directive('ducSshkeySwitch', sshKeySwitchDirective);
```
Same here, we should prefix and update usages in existing code.